examples/update.cpp

```diff
@@ -47,7 +47,7 @@ int main(int argc, char **argv) {
     stor->replace(Employee{ 7, "James", 24, "Houston", 10000.0 });
 
     //  show 'COMPANY' table contents
-    for(auto &employee : stor->iterate<Employee>()) {
+    for(auto&& employee : stor->iterate<Employee>()) {
```

Comment:
why do you use rvalue ref here?

Reply:
Ah - that's a vestige from when I was trying to get the `shared_ptr` out of `iterator_t`, before I realized that C++ doesn't currently alow move-only iterators. I'll back it out.

Reply:
Thanks. Once it is done I'll be ready to merge the PR

Reply:
Done! See 7304178